apps/photos/src/services/upload/uploader.ts

```diff
@@ -116,12 +116,13 @@ export default async function uploader(
         if (file.hasStaticThumbnail) {
             metadata.hasStaticThumbnail = true;
         }
-        let pubMagicMetadata: FilePublicMagicMetadata;
-        if (uploaderName) {
-            pubMagicMetadata = await uploadService.constructPublicMagicMetadata(
-                { uploaderName }
-            );
-        }
+
+        const pubMagicMetadata =
```

Comment:
If both uploadName and h/w are missing, will this return nil ?

Reply:
So, I was adding them as null. 

Have updated the code to ignore `null`/ `undefined` values from magicMetadata